nekoyume/Assets/_Scripts/BlockChain/ActionHandler.cs

```diff
@@ -150,7 +157,19 @@ public async UniTask UpdateCurrentAvatarStateAsync(AvatarState avatarState)
                 }
             }
 
+            UpdateCache(avatarState);
             await UpdateAvatarState(avatarState, States.Instance.CurrentAvatarKey);
         }
+
+        internal static void UpdateCombinationSlotState(int slotIndex, CombinationSlotState state)
+        {
+            States.Instance.UpdateCombinationSlotState(slotIndex, state);
+            UpdateCache(state);
+        }
+
+        private static void UpdateCache(Model.State.State state)
+        {
+            Game.Game.instance.CachedStates[state.address] = state.Serialize();
```

Comment:
Is it okay if don't check the key? 🤔 
`if (Game.Game.instance.CachedStates.ContainsKey(state.address))`